### ISSUE TITLE
Missing image alt texts

### ISSUE THREAD

**Issue by MattiasSp** [origo-map/origo#2360]
**Description**
Alt texts are missing on some image elements, such as for extendedlegend images. The EU directive WCAG 2.1 requires linked images to provide an alt text describing its purpose.

**To Reproduce**
1. Configure a layer to use a style with `"extendedLegend": true`
2. Inspect the resulting HTML

**Expected behavior**
Screen readers can read the aloud the purpose of the linked image through a provided alt text.

**Additional context**
Based on the Swedish law of accessibility of public digital services (DOS-lagen) the Digg government agency is now auditing websites and providing feedback where lacking accessibility is found. See their audit manual for further information: https://www.digg.se/kunskap-och-stod/digital-tillganglighet/tillsynsmanual-for-granskning-av-webbsidor#h-Lankar
